apps/src/util/hooks/useHiddenFileInput.tsx

```diff
@@ -0,0 +1,38 @@
+import React, {ChangeEvent, useRef} from 'react';
+
```

Comment:
Is it worth using this in the AI Chat UI as well, at some point?

Reply:
Yep! Will pull this into the AI Chat upload flow next (since there will be changes there to hook into this dialog anyway).